### PR TITLE
KYLIN-4999 Make Spark3.1 compatible with hadoop2.7 and hive1.2 in standalone docker image

### PR DESCRIPTION
**Pull request by MorningLight5**
## Proposed changes
JIRA: https://issues.apache.org/jira/browse/KYLIN-4999

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have create an issue on [Kylin's jira](https://issues.apache.org/jira/browse/KYLIN), and have described the bug/feature there in detail
- [ ] Commit messages in my PR start with the related jira ID, like "KYLIN-0000 Make Kylin project open-source"
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If this change need a document change, I will prepare another pr against the `document` branch
- [ ] Any dependent changes have been merged

### CODE REVIEW

File: docker/dockerfile/standalone/Dockerfile_hadoop [hunk 1]
```diff
@@ -19,8 +19,8 @@
 FROM centos:7.9.2009
 
 ENV HIVE_VERSION 1.2.1
-ENV HADOOP_VERSION 2.8.5
-ENV SPARK_VERSION 2.4.7
+ENV HADOOP_VERSION 2.7.7
+ENV SPARK_VERSION 3.1.1
```
Comment (xiacongling) [inline docker/dockerfile/standalone/Dockerfile_hadoop:23]: 可以用变量控制吗？默认用 2.4.7。通过打镜像的 .sh 脚本传入。

File: parquet-assembly/pom.xml [hunk 2]
```diff
@@ -113,6 +113,7 @@
                                         <exclude>META-INF/*.SF</exclude>
                                         <exclude>META-INF/*.DSA</exclude>
                                         <exclude>META-INF/*.RSA</exclude>
+                                        <exclude>org/slf4j/**</exclude>
```
Comment (xiacongling) [inline parquet-assembly/pom.xml:116]: 2.x 的镜像也简单测试一下吧，看移除了 sfl4j 会不会有问题

File: docker/dockerfile/standalone/Dockerfile_hadoop (hunk 1 above)
Comment (tianhui5) [reply docker/dockerfile/standalone/Dockerfile_hadoop:23]: stand alone镜像有必要支持多版本吗，如果要支持需要将tar.gz包也上传多个

File: parquet-assembly/pom.xml (hunk 2 above)
Comment (tianhui5) [reply parquet-assembly/pom.xml]: 已测试，slf4j可以不删

**Pull request closed without merging**